packages/tupaia-web/src/features/Map/MapTableModal/MapTableModal.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import DownloadIcon from '@material-ui/icons/GetApp';
+import MuiIconButton from '@material-ui/core/IconButton';
+import { FlexColumn } from '@tupaia/ui-components';
+import { Modal } from '../../../components';
+import { MapTable } from '@tupaia/ui-map-components';
+import { useParams } from 'react-router';
+import { useEntityAncestors } from '../../../api/queries';
+
+const Wrapper = styled(FlexColumn)`
+  justify-content: flex-start;
+`;
+
+const Title = styled.div.attrs({
+  variant: 'h2',
+})`
+  font-size: 1.25rem;
+  line-height: 1.4;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+  margin-bottom: 1.5rem;
+`;
+
+const IconButton = styled(MuiIconButton)`
+  margin-bottom: 1.5rem;
+`;
+
+const TitleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin-bottom: 3rem;
+  position: relative;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.12);
+`;
+
+export const MapTableModal = ({
+  serieses,
+  selectedOverlay,
+  processedMeasureData,
+  setIsOpen,
+}: any) => {
+  const handleCloseModal = () => {
+    setIsOpen(false);
+  };
+
+  const { projectCode, entityCode } = useParams();
+  const { data = [] } = useEntityAncestors(projectCode, entityCode);
+  const entityArray = data.reverse();
```

Comment:
Can you refresh my memory on why we need to reverse this?

Reply:
When `useEntityAncestors` creates it's array, it lists the members starting with deepest level at 0. Meaning that as you go deeper into the entity hierarchy (for example to village or region level, the actual country name gets further and further down the array. But if I reverse it, the country level remains at position 1. I'm creating this array as the map overlay table title should read <map overlay name>, <country>.